config-service-impl/gradle.lockfile

```diff
@@ -3,10 +3,14 @@
 # This file is expected to be part of source control.
 com.auth0:java-jwt:4.4.0=runtimeClasspath
 com.auth0:jwks-rsa:0.22.0=runtimeClasspath
-com.fasterxml.jackson.core:jackson-annotations:2.16.0=compileClasspath,runtimeClasspath
-com.fasterxml.jackson.core:jackson-core:2.16.0=compileClasspath,runtimeClasspath
-com.fasterxml.jackson.core:jackson-databind:2.16.0=compileClasspath,runtimeClasspath
-com.fasterxml.jackson:jackson-bom:2.16.0=compileClasspath,runtimeClasspath
+com.fasterxml.jackson.core:jackson-annotations:2.16.0=compileClasspath
+com.fasterxml.jackson.core:jackson-annotations:2.16.1=runtimeClasspath
+com.fasterxml.jackson.core:jackson-core:2.16.0=compileClasspath
+com.fasterxml.jackson.core:jackson-core:2.16.1=runtimeClasspath
+com.fasterxml.jackson.core:jackson-databind:2.16.0=compileClasspath
+com.fasterxml.jackson.core:jackson-databind:2.16.1=runtimeClasspath
+com.fasterxml.jackson:jackson-bom:2.16.0=compileClasspath
+com.fasterxml.jackson:jackson-bom:2.16.1=runtimeClasspath
```

Comment:
This BOM mismatch indicates some dependency is ahead of the BOM, which ideally we don't want (since that means we're running with a different version than we're compiling with). Looks like it's coming from doc store. @suresh-prakash do you mind bringing the BOM up to 2.16.1 to match?

Reply:
https://github.com/Traceableai/traceable-bom/pull/93

Reply:
Hypertrace BOM please

Reply:
My bad. https://github.com/hypertrace/hypertrace-bom/pull/27